packages/next-adapter/src/withExpo.ts

```diff
@@ -1,9 +1,8 @@
 import { getBareExtensions } from '@expo/config/paths';
 import { withUnimodules } from '@expo/webpack-config/addons';
 import { AnyConfiguration } from '@expo/webpack-config/webpack/types';
-import { NextConfig } from 'next';
 
-export default function withExpo(nextConfig: NextConfig = {}): NextConfig {
+export default function withExpo(nextConfig: any = {}): any {
```

Comment:
lol ok

Reply:
Can change it in next PR to `Record<string, any>` if you want, but probably we cannot go more strict than that, without writing the types by hand. 😉